index.html

```diff
@@ -22,8 +22,7 @@
             <h1>
               Say Hello World to Event Streaming.
             </h1>
-            <h2 class="subtitle">Apache Kafka is a powerful, scalable, fault-tolerant distributed streaming platform. This site features full code examples using Kafka, Kafka Streams, and ksqlDB to demonstrate real use cases.</h2>
-            <h4>All the tutorials can be run locally or with <a href="https://www.confluent.io/confluent-cloud/tryfree/">Confluent Cloud</a>, Apache Kafka® as a fully managed cloud service.</h4>
+            <h2 class="subtitle">This site features full code examples using Apache Kafka®, Kafka Streams, and ksqlDB to demonstrate real use cases. The examples can be run locally or with <a href="https://www.confluent.io/confluent-cloud/tryfree/">Confluent Cloud</a>, Apache Kafka as a fully managed cloud service, deployable on<br><a href="https://www.confluent.io/confluent-cloud/tryfree/"><img src="{{ "/assets/img/providers.png" | relative_url }}" width="200" /></a></h2>
```

Comment:
One minor nit - can we get `deployable on..` and the icons all on the same line? Right now the logos break on a separate line and IMHO it's slightly harder to follow.

![Screen Shot 2021-08-09 at 11 25 12 AM](https://user-images.githubusercontent.com/199238/128732250-8cf02e37-1bf3-4c35-93ed-aa072a692b22.png)

Reply:
@bbejeck 👍 trying it out in latest commit